examples/classification/classification/evaluator_binary.py

```diff
@@ -123,6 +125,21 @@ def compute_test_case_plots(
                 labels=[positive_label, negative_label],
             ),
         )
+
+        threshold_curves = compute_threshold_curves(
```

Comment:
the [return type](https://github.com/kolenaIO/kolena/blob/15b2ae3c976074197ea26e40214d2e46cd60d656/kolena/_experimental/classification/utils.py#L263C9-L263C9) of `compute_threshold_curves` is slightly off, but rest looks good!

Reply:
`Optional[List[CurvePlot]]` -> `Optional[List[Curve]]`

Reply:
OH! 🙈